src/main/java/jp/co/future/uroborosql/SqlAgent.java

```diff
@@ -37,7 +39,7 @@
 	 * 文字列として{@link CoverageHandler}インタフェースの実装クラスが設定された場合はそのクラスを<br>
 	 * 利用してカバレッジの収集を行う。
 	 */
-	final String KEY_SQL_COVERAGE = "uroborosql.sql.coverage";
+	String KEY_SQL_COVERAGE = "uroborosql.sql.coverage";
```

Comment:
Why did you delete final?

Reply:
Inserface does not need to have `final` modifire.

Reply:
Certainly Well